apps/webapp/app/send/types.ts

```diff
@@ -0,0 +1,18 @@
+export enum SendPageTab {
+  SEND = 'send',
+  RECEIVE = 'receive',
+  IBC = 'ibc',
+}
+
+export type TabHelper = Record<
+  SendPageTab,
+  {
+    src: string;
+    label: string;
+  }
+>;
+
+export interface Asset {
+  name: string;
+  icon: string;
+}
```

Comment:
As an exported type `Asset` name is too generic as it relates only to this page

Reply:
this is test data, in future we'll have another types